k8s/kube-base/job-test.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: batch/v1
```

Comment:
@rob-baron  I don't think you need this `Job`; you can [spawn a Job from a CronJob manually](https://www.craftypenguins.net/how-to-trigger-a-kubernetes-cronjob-manually/) using e.g.:

```
kubectl create job --from=cronjob/xdmod-ingestor test-xdmod-ingestor
```

Reply:
I included this for Mainn, as it is a way to just test the job.  will delete.

Reply:
I've removed that commit from the PR.